packages/dds/sequence/README.md

```diff
@@ -245,6 +276,122 @@ As mentioned above, annotate operations behave like operations on SharedMaps. Th
+
+
+```typescript
+    //   content: hi world!
+    // positions: 012345678
+
+    const comments = sharedString.getIntervalCollection("comments");
+    const comment = comments.add(
+        3,
+        7, // (inclusive range): references "world"
+        IntervalType.SlideOnRemove,
+        {
+            creator: 'my-user-id',
+            handle: myCommentThreadDDS.handle
+        }
+    );
+    //   content: hi world!
+    // positions: 012345678
+    //   comment:    [   ]
+
+    // Interval collection supports iterating over all intervals via Symbol.iterator or `.map()`:
+    const allIntervalsInCollection = Array.from(comments);
+    const allProperties = comments.map((comment) => comment.properties);
+    // or iterating over intervals overlapping a region:
+    const intervalsOverlappingFirstHalf = comments.findOverlappingIntervals(0, 4);
+
+    // Interval endpoints are LocalReferencePositions, so all APIs in the above section can be used:
+    const startPosition = sharedString.localReferencePositionToPosition(comment.start);
+    const endPosition = sharedString.localReferencePositionToPosition(comment.end);
+
+    // Intervals can be modified:
+    comments.change(comment.getIntervalId(), 0, 1);
+    //   content: hi world!
+    // positions: 012345678
+    //   comment: []
+
+    // their properties can be changed:
+    comments.changeProperties(comment.getIntervalId(), { status: "resolved" });
+    // comment.properties === { creator: 'my-user-id', handle: <some DDS handle object>, status: "resolved" }
+
+    // and they can be removed:
+    comments.removeIntervalById(comment.getIntervalId());
+```
+
+### Intervals vs. Markers
+
+Interval endpoints and markers both implement `ReferencePosition` and seem to serve a similar function;
+a reasonable question to ask is how they are different to a user, and why you might want to use one or the other.
+
+Using the interval collection API has two main benefits:
+
+1. Efficient spatial querying
+    - Interval collections support iterating all intervals overlapping the region `[start, end]` in `O(log N) + O(overlap size)` time, where `N` is the total number of intervals in the collection.
+    This may be critical for applications that display only a small view of the document contents.
+    On the other hand, using markers to implement intervals would require a linear scan from the start or end of the string to determine which intervals overlap.
+
+2. Reduced risk of data loss on modification
+    - Interval collections natively support a `modify` operation on the intervals, which allows moving the endpoints of the interval to a different place in the sequence.
+    This operation is atomic, whereas with markers one would have to submit a `delete` operation for the existing position and an `insert` for the new one.
+    This has some small potential for data loss if the delete operation ends up sequenced but the insert does not.
```

Comment:
using "acknowledged" but otherwise similar phrasing seems just as accurate and should hopefully be more understandable (totally fair point I didn't think of.. definitely better to avoid overloading the term :))